lib/index.js

```diff
@@ -179,6 +200,39 @@ FSTree.defaultIsEqual = function defaultIsEqual(entryA, entryB) {
   return equal;
 };
 
+FSTree.applyPatch = function(input, output, patches, _delegate) {
```

Comment:
@trentmwillis that works.

We also have the opportunity to rename these, but we can chat about that at a later time.